tests/tensor/test_basic.py

```diff
@@ -7388,21 +7388,29 @@ def test_dtype_error(self):
         with pytest.raises(TypeError):
             choose(a, b)
 
-    def test_numpy_compare_tuple(self):
-
-        a = tt.tensor3(dtype="int32")
-        b = tt.tensor3(dtype="float32")
-        c = tt.tensor3(dtype="float32")
-
-        A = np.random.randint(0, 2, (2, 1, 1)).astype("int32")
-        B = np.asarray(np.random.rand(1, 6, 1), dtype="float32")
-        C = np.asarray(np.random.rand(1, 1, 5), dtype="float32")
+    a1 = tt.tensor3(dtype="int32")
+    b1 = tt.tensor3(dtype="float32")
+    c1 = tt.tensor3(dtype="float32")
+    A1 = np.random.randint(0, 2, (2, 1, 1)).astype("int32")
+    B1 = np.asarray(np.random.rand(1, 6, 1), dtype="float32")
+    C1 = np.asarray(np.random.rand(1, 1, 5), dtype="float32")
+    a2 = tt.vector(dtype="int32")
+    b2, c2 = tt.scalar(), tt.scalar()
+    A2 = [0, 1, 1, 0]
+    B2, C2 = 0.1, 0.2
```

Comment:
These shouldn't be defined at class-level unless that's needed; if not, these values should be created right in the `pytest.mark.parameterize` tuples below.

Reply:
This has been integrated into the initial commit.